<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTcpClient</class>
 <widget class="QWidget" name="frmTcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="1">
    <widget class="QPushButton" name="btnSend">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Things</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="txtMain">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="Publish">
     <property name="text">
      <string>Publish</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="Subscribe">
     <property name="text">
      <string>Subscribe</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_11">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Publish</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="S_TOPIC_NAME">
     <property name="text">
      <string>/sys/a1yLNNi0bK0/D001/thing/service/property/set</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Subscribe</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="layTcpClient">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cboxData">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="maxVisibleItems">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2" rowspan="9">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QCheckBox" name="ckHexReceive">
        <property name="text">
         <string>16进制接收</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckHexSend">
        <property name="text">
         <string>16进制发送</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckAscii">
        <property name="text">
         <string>Ascii控制字符</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckShow">
        <property name="text">
         <string>暂停显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckDebug">
        <property name="text">
         <string>模拟设备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckAutoSend">
        <property name="text">
         <string>定时发送</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboxInterval"/>
      </item>
      <item>
       <widget class="QLabel" name="labServerIP">
        <property name="text">
         <string>服务器IP地址</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtServerIP">
        <property name="text">
         <string>a1yLNNi0bK0.iot-as-mqtt.cn-shanghai.aliyuncs.com</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labServerPort">
        <property name="text">
         <string>服务器端口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtServerPort">
        <property name="text">
         <string>1883</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConnect">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>连接</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ProductKey</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ProductKey">
        <property name="text">
         <string>a1yLNNi0bK0</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>DeviceName</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DeviceName">
        <property name="text">
         <string>D001</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>DeviceSecret</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="DeviceSecret">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>nDxnGo3XhfE5lfhDdVlGtaPFIfQ3F3Z8</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>连接阿里云</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="P_TOPIC_NAME">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>/sys/a1yLNNi0bK0/D001/thing/event/property/post</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="Things">
     <property name="text">
      <string>{&quot;method&quot;:&quot;thing.event.property.post&quot;,&quot;id&quot;:&quot;203302322&quot;,&quot;params&quot;:{&quot;EnvironmentHumidity&quot;:50.00,&quot;CurrentTemperature&quot;:30.30},&quot;version&quot;:&quot;1.0.0&quot;}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
